macros/filter_bots.sql

```diff
@@ -0,0 +1,23 @@
+{% macro filter_bots() %}
```

Comment:
Shouldn't we allow users to specify bot strings they want to check? How does this enable custom bot filtering @agnessnowplow?

Reply:
Also I'd expect we would introduce something in the docs to show how this works within this same commit, right?

Reply:
Nvm, here's the docs PR which outlines it!
https://github.com/snowplow/documentation/pull/144